api/spec/services/validator/html_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+RSpec.describe Validator::Html do
+
+  let(:validator) { Validator::Html.new() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.